pyomo/core/base/instance2dat.py

```diff
@@ -29,21 +29,21 @@ def instance2dat(instance, output_filename):
             if set_object.dim() == 0:
                 if len(set_object) == 0:
                     continue
-                print >>output_file, "set "+set_name+" := "
+                output_file.write("set "+set_name+" := \n")
                 for element in set_object:
-                    print >>output_file, element
-                print >>output_file, ";"
+                    output_file.write(element,)
+                output_file.write(";\n")
             elif set_object.dim() == 1:
                 for index in set_object:
-                    print >>output_file, "set "+set_name+"[\""+str(index)+"\"]"+" := ",
+                    output_file.write("set "+set_name+"[\""+str(index)+"\"]"+" := \n")
```

Comment:
This `\n` should be removed (the trailing `,` in the original file suppressed the `\n`)